src/components/explanation/CfFeedbackResulttTable.js

```diff
@@ -0,0 +1,114 @@
+import React, {PureComponent} from 'react';
+import {DataTable, TableBody, TableColumn, TableHeader, TablePagination, TableRow} from 'react-md/lib/DataTables/index';
+import PropTypes from 'prop-types';
+import {getPatternsForMatrix, getConfusionMatrixLabels} from '../../util/dataReducers';
+import {SelectField} from 'react-md';
+
+class CfFeedbackResulttTable extends PureComponent {
```

Comment:
I guess this is the source file with the typo CfFeedbackResulttTable -> CfFeedbackResultTable